ibmcloudant/cloudant_base_service.py

```diff
@@ -160,7 +160,9 @@ def _error_response_hook(response:Response, *args, **kwargs) -> Optional[Respons
                             error_json['errors'] = [error_model]
                             send_augmented_response = True
                     if 'errors' in error_json:
-                        trace = response.headers.get('x-couch-request-id')
+                        trace = response.headers.get('x-request-id')
+                        if trace is None:
+                            trace = response.headers.get('x-couch-request-id')
```

Comment:
You could also do something like below although it's not as clear:
```
trace = response.headers.get('x-request-id', response.headers.get('x-couch-request-id'))
```

Reply:
Done in df0acb13787097352a8d3a7353287150d8293a29 with a comment so hopefully it is as clear 😄